auth/AuthController.js

```diff
@@ -1,47 +1,40 @@
 const express = require('express');
 const bodyParser = require('body-parser');
-const VerifyToken = require('../VerifyToken');
 const jwt = require('jsonwebtoken');
 const bcrypt = require('bcryptjs');
 const uuidgen = require('node-uuid-generator');
-const APIUser = require('../data/models/APIUser');
+const Database = require('../data/database');
 const logger = require('../logger');
 
 const router = express.Router();
 router.use(bodyParser.urlencoded({ extended: false }));
 router.use(bodyParser.json());
+const database = new Database();
 
-router.post('/register', async function(req, res) {
+router.post('/token', async function(req, res) {
   try {
-    logger.info(`/register ${req.body.name}`);
+    logger.info(`/register ${req.body.userid}`);
```

Comment:
Change the logging to /token instead of /register

Reply:
fixed